packages/vats/src/core/manifest.js

```diff
@@ -82,6 +82,14 @@ const SHARED_CHAIN_BOOTSTRAP_MANIFEST = harden({
     },
     home: { produce: { chainTimerService: 'timer' } },
   },
+  makeChainStorage: {
+    consume: {
+      bridgeManager: true,
+    },
+    produce: {
+      chainStorage: true,
```

Comment:
This produces chainStorage, but there is not currently anything to consume it. The biggest open question is "how should a derived capability be made available to vats—what makes new child nodes, and where do they get sent?" Neither @warner nor I could find an existing analog, but perhaps the board comes closest?

Some possible answers:
* Zoe has access to the root node, and issues a child node of a common parent to contract instances that are somehow marked as privileged.
* Something in the bootstrap code has access to the root node, and issues descendant nodes as directed by manifest data (either as part of startInstance or after the fact).
* Other post-chain-initialization code has access to the root node, and issues descandant nodes as remoteables to already-running contract instances.

Reply:
Summarising what I understood from the most recent conversation:

- We start by bootstrap having access to the root node, as you have now, on which we will publish (via this publication mechanism) a name-to-storage-path mapping.
- Governance "core eval" code will issue descendants to certain contracts via Zoe private arguments and update the name-to-storage path mapping.
- We will use explicit ocap introduction patterns to obtain descendants and descendants of descendants outside bootstrap.
- There's no pressing need for automatic allocation of descendants.  Indeed, that's probably an antipattern until we're certain of performance characteristics.